packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss.vue

```diff
@@ -37,52 +37,53 @@
 						/>
 					</div>
 				</div>
-				<div class="form-section">
+				<section class="form-section">
 					<h5>
 						Intervention policy
 						<i v-tooltip="interventionPolicyToolTip" class="pi pi-info-circle" />
 					</h5>
-					<div>
-						<label>Intervention Type</label>
-						<Dropdown
-							class="p-inputtext-sm"
-							:options="[
-								{ label: 'parameter value', value: InterventionTypes.paramValue },
-								{ label: 'start time', value: InterventionTypes.startTime }
-							]"
-							option-label="label"
-							option-value="value"
-							v-model="knobs.interventionType"
-							placeholder="Select"
-						/>
-					</div>
+					<!--					<div>-->
+					<!--						<label>Intervention Type</label>-->
+					<!--						<Dropdown-->
+					<!--							class="p-inputtext-sm"-->
+					<!--							:options="[-->
+					<!--								{ label: 'parameter value', value: InterventionTypes.paramValue },-->
+					<!--								{ label: 'start time', value: InterventionTypes.startTime }-->
+					<!--							]"-->
+					<!--							option-label="label"-->
+					<!--							option-value="value"-->
+					<!--							v-model="knobs.interventionType"-->
+					<!--							placeholder="Select"-->
+					<!--						/>-->
+					<!--					</div>-->
 					<tera-intervention-policy-group-form
-						v-for="(cfg, idx) in props.node.state.interventionPolicyGroups"
+						v-for="(cfg, idx) in interventions"
 						:key="idx"
 						:config="cfg"
+						:knobs="knobs"
 						:intervention-type="props.node.state.interventionType"
 						:parameter-options="modelParameterOptions.map((ele) => ele.id)"
 						@update-self="(config) => updateInterventionPolicyGroupForm(idx, config)"
 						@delete-self="() => deleteInterverntionPolicyGroupForm(idx)"
 					/>
-					<div>
-						<Button
-							icon="pi pi-plus"
-							class="p-button-sm p-button-text"
-							label="Add more interventions"
-							@click="addInterventionPolicyGroupForm"
-						/>
-					</div>
-				</div>
-				<div class="form-section">
+					<!--					<div>-->
```

Comment:
Safe to remove this as we are no longer allowed to add more interventions with new design